tx_pool/src/pool.rs

```diff
@@ -150,6 +150,79 @@ impl Pool {
         self.update_order_set(txs);
     }
```

Comment:
so, we can remove the `package` method ?

Reply:
we can but not now.  remove when cita-auth modify the use of package

Reply:
why ?

Reply:
cita-auth can't compile success without package method now